src/java.base/share/classes/jdk/internal/classfile/impl/ClassRemapperImpl.java

```diff
@@ -273,7 +273,6 @@ public CodeTransform asCodeTransform() {
 
     @Override
     public ClassDesc map(ClassDesc desc) {
-        if (desc == null) return null;
```

Comment:
I'm not sure this won't break some use case.
Here I would say null -> null mapping might be OK.

Reply:
Makes sense. I will push the changes when you finish the review.

Reply:
It is all I have.